public/api/docs/v3/swagger.yaml

```diff
@@ -214,7 +214,7 @@ paths:
                         teacher_catchment_iso_country_code: GBR
                         itt_provider: University of Southampton
                         lead_mentor: true
-                        funded_place:
+                        funded_place: 
```

Comment:
Odd how this is adding the whitespace on the end of lines - I think these whitespace only changes can be skipped